blockmanager.go

```diff
@@ -1200,19 +1691,23 @@ func (b *blockManager) handleHeadersMsg(hmsg *headersMsg) {
 		if err != nil {
 			log.Warnf("Failed to send getheaders message to "+
 				"peer %s: %s", hmsg.peer.Addr(), err)
-			// Unnecessary but we might put other code after this
-			// eventually.
 			return
 		}
 	}
+
+	// If we've caught up to the headers, we can start the cfheader sync
+	if b.current() && b.startCFHeaderSync != nil {
+		close(b.startCFHeaderSync)
+		b.startCFHeaderSync = nil
+	}
 }
 
 // sendGetcfheaders sends a getcfheaders message to all peers with the
 // specified start block and hash block and for the specified type of header.
 func (b *blockManager) sendGetcfheaders(startHeight uint32, endBlock *chainhash.Hash,
 	count int, filterType wire.FilterType) {
 	// Send getcfheaders to each peer based on these headers.
-	cfhReq := cfhRequest{
+	/* cfhReq := cfhRequest{
```

Comment:
Disconnected sending the `getcfheaders` request to peers here as well so it doesn't interfere with the query-based wiring.